### PR TITLE
Updates number of events per page to 30

### PR DESCRIPTION
**Pull request by railto**
Closes #JOINDIN-592

### CODE REVIEW

**Comment by akrabat** [comment]
ok to test

File: app/src/Event/EventController.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 class EventController extends BaseController
 {
-    private $itemsPerPage = 10;
+    private $itemsPerPage = 30;
```
Comment (akrabat) [inline app/src/Event/EventController.php]: JOINDIN-592 is for the pending events list page, not every events list page.

Comment (railto) [inline app/src/Event/EventController.php]: Dang, correcting

**Comment by railto** [comment]
Cancelling PR per conversation with @lornajane

**Pull request closed without merging**